### PR TITLE
fix(DB/Loot): Orphic Bracers

### PR DESCRIPTION
**Pull request by Synful-Syn**
## Changes Proposed:
-  Remove `Orphic Bracers` from the `reference_loot_template` which is used by 746 `creature_loot_template`.
-  Created the 4 specific `creature_loot_template` rows for the `Orphic Bracers`.

## Issues Addressed:

- Closes nothing, there were no reports.

## SOURCE:

They all shows the same 4 mobs.
Classic Vanilla WowHead: https://www.wowhead.com/classic/item=18337/orphic-bracers
Classic TBC WowHead: https://www.wowhead.com/tbc/item=18337/orphic-bracers
Classic Wrath WowHead: https://www.wowhead.com/wotlk/item=18337/orphic-bracers
WoWWiki: https://wowwiki-archive.fandom.com/wiki/Orphic_Bracers
Wowpedia: https://wowpedia.fandom.com/wiki/Orphic_Bracers

## Tests Performed:

- 

## How to Test the Changes:

1. Grind!

## Known Issues and TODO List:

- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/fix-orphic-bracers-loot.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DELETE FROM reference_loot_template
+WHERE Item = 18337;
+
+INSERT INTO creature_loot_template (Entry, Item, Reference, Chance, QuestRequired, LootMode, GroupId, MinCount, MaxCount, Comment) VALUES
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/fix-orphic-bracers-loot.sql:4]: remember the backticks (`)
```suggestion
DELETE FROM `reference_loot_template` WHERE `Item` = 18337;
INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
```

File: data/sql/updates/pending_db_world/fix-orphic-bracers-loot.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+DELETE FROM `reference_loot_template` WHERE `Item` = 18337;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/fix-orphic-bracers-loot.sql:2]:
```suggestion
INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
```

File: data/sql/updates/pending_db_world/fix-orphic-bracers-loot.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
+DELETE FROM `reference_loot_template` WHERE `Item` = 18337;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/fix-orphic-bracers-loot.sql:2]: I hadn't noticed it before.
Missing DELETE before INSERT in the `creature_loot_template` table

Comment (Synful-Syn) [reply data/sql/updates/pending_db_world/fix-orphic-bracers-loot.sql]: It has been added! :D

**Review by Gultask — APPROVED** [review]

**Pull request merged**